cubic-server/Entity.hpp

```diff
@@ -72,6 +73,8 @@ class Entity : public utility::SharedFromThis<Entity> {
     NODISCARD virtual const Vector3<double> &getPosition() const;
     NODISCARD virtual Vector2<uint8_t> &getRotation();
     NODISCARD virtual const Vector2<uint8_t> &getRotation() const;
+    NODISCARD virtual Vector2<float> getRotationDegree();
+    NODISCARD virtual const Vector2<float> getRotationDegree() const;
```

Comment:
Please return a reference for those two functions

Reply:
Those functions return a locally declared variable, can't reference that :pepe_sad:

Reply:
Then just create only one function like this:
```suggestion
    NODISCARD virtual Vector2<float> getRotationDegree() const;
```